src/internal/components/configurationPanels/HeatMapConfigurationPanel.tsx

```diff
@@ -45,6 +46,13 @@ export default class HeatMapConfigurationPanel extends ConfigurationPanelContent
                         properties={properties}
                         pushData={pushData}
                     >
+                        <NameSubsection
+                            disabled={xAxisDisabled}
+                            configPanelDisabled={controlsDisabled}
+                            axis={"xaxis"}
+                            properties={properties}
+                            pushData={pushData}
+                        />
```

Comment:
share some properties with NameSubsection

Reply:
I think we should not group it